content/source/docs/cloud/registry/publish.html.md

```diff
@@ -60,15 +60,15 @@ that don't look like version numbers are ignored.
 
 ## Publishing a New Module
 
-To publish a module, navigate to the modules list with the "Modules" button and click the "+ Add Module" button in the upper right.
+To publish a module, navigate to the modules list with the "Registry" button and click the "Publish" button in the upper right and then the "Module" button that appears under it.
 
-![Terraform Cloud screenshot: the "modules" button and the "+Add Module" button](./images/publish-add-button.png)
+![Terraform Cloud screenshot: the "registry" button and the "+Add Module" button](./images/publish-add-button.png)
```

Comment:
This picture doesn't have Registry in it - I think it should be in the main navigation bar, right?

Reply:
Yes, it is under the Registry menu.  I just pushed new image that has that.